cpp/ps0001/ps0001_soln.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+  // prompting user for 3 numbers to add 1.5/2
+  unsigned short num1, num2, num3;
+  cout << "please enter first number \n";
+  cin >> num1;
+  cout << "please enter second number \n";
+  cin >> num2;
+  cout << "please enter thired number \n";
+  cin >> num3;
+
+  // outputting the result of the summation process 3/3
+  cout << "the result is\t" << num1 + num2 + num3 << "\n";
```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
+1 bonus mark as your program is user-friendly